tests/foreman/api/test_role.py

```diff
@@ -218,8 +318,30 @@ def test_positive_create_overridable_filter(self):
 
         :CaseImportance: Critical
         """
+        role_name = gen_string('alpha')
+        role1 = entities.Role(
+            name=role_name,
+            organization=[self.org1],
+            location=[self.loc1]
+        ).create()
+        self.assertEqual(role1.name, role_name)
+        dom_perm = (entities.Permission(resource_type='Domain').search())
+        filter1 = entities.Filter(
+            permission=dom_perm,
+            role=role1.id,
+            override=True,
+            organization=[self.org2],
+            location=[self.loc2]
+        ).create()
+        self.assertEqual(role1.id, filter1.role.id)
+        filter1_orgs = filter1.organization
+        filter1_locs = filter1.location
```

Comment:
What's the point of these vars? You can use filter1.organization or filter1.location directly

Reply:
There's rather a point in extra var for list of orgs/locs correspondingly - not to generate it for the second time. I mean
```
filter1_orgs = [org.id for org in filter1.organization]

Reply:
Yeah I used it two times below and that also increases the length hence.
But I updated it without any length issue :)